src/io/flutter/run/daemon/DaemonConsoleView.java

```diff
@@ -81,12 +81,6 @@ public void print(@NotNull String text, @NotNull ConsoleViewContentType contentT
       return;
     }
 
-    // "flutter test" prints this when using --start-paused.
-    // (We should probably change it to stop printing it in machine mode.)
-    if (trimmed.startsWith("You should first set appropriate breakpoints, then resume the test in the debugger.")) {
-      return;
-    }
-
```

Comment:
Since we're shooting for a release next week (and can't ensure they're on alpha besides), is there an easy way to make this fail gracefully on unsupported SDKs?